web/src/components/TestSpecForm/SelectorInput.tsx

```diff
@@ -0,0 +1,28 @@
+import {Form, FormInstance} from 'antd';
+import {IValues} from './TestSpecForm';
+import AdvancedEditor from '../AdvancedEditor';
+import useQuerySelector from './hooks/useQuerySelector';
+
+interface IProps {
+  form: FormInstance<IValues>;
```

Comment:
We can create a custom type for `FormInstance<IValues>` to be used across the child inputs